packages/key-management/src/types.ts

```diff
@@ -38,10 +38,17 @@ export enum KeyRole {
   DRep = 3
 }
 
+export enum KeyPurpose {
+  STANDARD = 1852,
+  MULTI_SIG = 1854
+}
+
 export interface AccountKeyDerivationPath {
   role: KeyRole;
   index: number;
+  purpose: KeyPurpose;
```

Comment:
This type is used to derive keys from account key that already has `purpose` derived
```suggestion
```

Reply:
Fixed on 7c3a32f